website/src/app/kb/authenticate/readme.mdx

```diff
@@ -5,3 +5,17 @@ Firezone supports the following authentication methods:
 1. [Email (OTP)](/kb/authenticate/email)
 1. [OpenID Connect (OIDC)](/kb/authenticate/oidc)
 1. [Google Workspace](/kb/authenticate/google)
+
+## Session lifetime
+
+The below session lifetimes apply to all identity providers:
+
+- Admin portal: **4 hours**
+- Client application: **7 days**
```

Comment:
I noticed you mentioned in the PR description that ideally all lifetimes would be configurable.  Is that something that's on the roadmap?  If so, would it make sense to add a small note saying these will be configurable in the future?

Reply:
It's not specifically listed on the public roadmap, but there's an issue for it here: #3179 

> would it make sense to add a small note saying these will be configurable in the future?

Hmm, as a general approach I've been trying to keep the future state out of the docs unless we want to solicit input on it. For this one I think it's mostly well-defined, just need to prioritize it